pyspectral/tests/test_utils.py

```diff
@@ -23,11 +23,14 @@
 
 """Do the unit testing for the utils library."""
 
-from pyspectral import utils
-
+import pytest
 import unittest
 import numpy as np
 
+from pyspectral import utils
+from pyspectral.utils import np2str, bytes2string
```

Comment:
I think the functions that are tested should be imported inside the test function/method. That way they can be mocked in other tests, if it becomes necessary.

Reply:
Okay, I thought the mocking was independent of the imports actually!?
I always have all imports at the top.